### PR TITLE
Merge sprint10 06 05 24

### PR DESCRIPTION
**Pull request by Daniielmayo**

### CODE REVIEW

**Review by SthefaniaEcheverry — APPROVED** [review]

**Review by abdulflorez — APPROVED** [review]

File: components/organisms/AboutUs/AboutUs.module.css [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+    flex-direction: column;
+    margin: 1.875rem;
+}
+
+.aboutUs__ourObjetiveParagraph2--text1 {
+    margin-bottom: 1.25rem;
+    line-height: 1.4rem;
+    font-size: 0.875rem;
+    text-align: justify;
+    color: #2f2e2e;
```
Comment (DanielCalleCO) [inline components/organisms/AboutUs/AboutUs.module.css:103]: Should this color be saved in a variable?

File: components/organisms/AboutUs/AboutUs.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
+"use client";
+import styles from "./AboutUs.module.css";
+
+export const AboutUs = () => {
+    const imagenLincolns = "https://static.wixstatic.com/media/d166cc_511a94f7936d4713b4906a3194d7f30c~mv2.jpg/v1/crop/x_283,y_60,w_994,h_1020/fill/w_166,h_172,al_c,q_80,usm_0.66_1.00_0.01,enc_auto/30743886_2126087857670517_61279119260652.jpg";
```
Comment (DanielCalleCO) [inline components/organisms/AboutUs/AboutUs.tsx:5]: Should this const be defined in english?

File: components/organisms/AboutUs/AboutUs.tsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+    const imagenAbigailMichell = "https://static.wixstatic.com/media/d166cc_141257b4382346d3a80798efa1cf09c4~mv2.jpg/v1/crop/x_129,y_49,w_440,h_521/fill/w_134,h_156,al_c,q_80,usm_0.66_1.00_0.01,enc_auto/54517632_2414760852078060_82520352340574.jpg";
```
Comment (DanielCalleCO) [inline components/organisms/AboutUs/AboutUs.tsx:7]: could you please translate this to english?

File: components/organisms/AboutUs/AboutUs.tsx [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+    const imagenDanielaLopez = "https://static.wixstatic.com/media/d166cc_065bdcc8edf34430a0341c1ba9fd48d7~mv2.jpg/v1/crop/x_219,y_0,w_2586,h_3132/fill/w_142,h_172,al_c,q_80,usm_0.66_1.00_0.01,enc_auto/Milka%20Daniela.jpg";
```
Comment (DanielCalleCO) [inline components/organisms/AboutUs/AboutUs.tsx:9]: could you please translate this to english?

File: components/organisms/AboutUs/AboutUs.tsx [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+    const discoverBible = "https://static.wixstatic.com/media/d166cc_fc0265a7548b4bc9b50511322f17870d~mv2.png/v1/fill/w_124,h_124,al_c,q_85,usm_0.66_1.00_0.01,enc_auto/3000PX%20Logos_Mesa%20de%20trabajo%201.png";
+
+    const imagenFooter = "https://static.wixstatic.com/media/d166cc_89e082e6997946c1924000b082e3d4ad~mv2.jpg/v1/fill/w_980,h_245,al_c,q_80,usm_0.66_1.00_0.01,enc_auto/d166cc_89e082e6997946c1924000b082e3d4ad~mv2.jpg";
```
Comment (DanielCalleCO) [inline components/organisms/AboutUs/AboutUs.tsx:13]: could you please translate this to english?

**Review by DanielCalleCO — APPROVED** [review]

**Pull request merged**